app/models/basic_project.rb

```diff
@@ -44,7 +44,21 @@
 #
 
 class BasicProject < Project
+  attr_option_settable :status
+
+  def start_date
+    signing_date
+  end
+
   def default_name
     I18n.t("common.untitled")
   end
+
+  def display_name
+    name.empty? ? default_name : name
```

Comment:
If `name` is `nil`, this will return `name`. `name.empty?` is `false`. You probably want `blank?`?